common/religions/necromancy_group.txt

```diff
@@ -135,4 +135,68 @@ necromancy_group = {
 
 		dynamic_cult = no	# Warcraft - can't handle it yet
 	}	
+	throsic = {
+		graphical_culture = westerngfx
+
+		icon = 91
+		heresy_icon = 92
+		
+		color = { 5 105 120 }
+		
+		scripture_name = NECRONOMICON
+		priest_title = NECROMANCER
+		piety_name = IMPIETY
+		
+		high_god_name = THROS
+		
+		god_names = {
+			THROS THE_BLIGHT THE_NECROMANCY
+		}
+		evil_god_names = {
+			THE_LIGHT THE_LIFE
+		}
+		
+		# Modifiers
+		aggression = 0.75
```

Comment:
They're less aggressive than the Holy Light? Default aggression is 1.

Reply:
Good catch. Fixed.